ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/ldesfragment/services/FragmentationQueueMediatorImpl.java

```diff
@@ -30,8 +35,12 @@ public boolean queueIsEmtpy() {
 		return ldesMembersToFragment.isEmpty();
 	}
 
-	@Scheduled(fixedDelay = 60000)
-	private void reportWaitingMembers() {
-		logger.info("Number of Members queued for fragmentation:\t {}", ldesMembersToFragment.size());
+	@Scheduled(fixedDelay = 1000)
+	protected void reportWaitingMembers() {
+		ldesMembersToFragmentTracker.set(ldesMembersToFragment.size());
+		// TODO open discussion whether we also want to keep the logging together with
+		// the metric exporter
+		// logger.info("Number of Members queued for fragmentation:\t {}",
+		// ldesMembersToFragment.size());
```

Comment:
opinion on this please

Reply:
I don't why this code is here, but it looks like debug logging. If that's the case, it can be removed in favour of metrics.